downstream/modules/dev-guide/proc-downloading-base-ees.adoc

```diff
@@ -0,0 +1,28 @@
+
+
+[id="downloading-base-ees"]
+
+
+
+= Downloading base {ExecEnvName}
+
+[role="_abstract"]
+Base images that ship with AAP 2.0 are hosted on the Red Hat Ecosystem Catalog (registry.redhat.io)
+
+.Prerequisites
+
+* You have a valid {PlatformName} subscription.
+
+.Procedure
+
+. Log in to registry.redhat.io
++
+-----
+$ podman login registry.redhat.com
+-----
++
+. Pull the base images from the registry
++
+-----
+$ podman pull registry.redhat.io/aap/<image name>
```

Comment:
Are we using italics to represent user-replaced values?
https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values

```
[subs="+quotes"]
-----
$ podman pull registry.redhat.io/aap/__<image_name>__
----
```

Reply:
updated

Reply:
The code in the update builds as 
registry.redhat.io/aap/_<image name>_

If you think we do need the italics for the replaceable value, <image_name> needs 2 underscore characters on either side, and the command needs [subs="+quotes"] on the preceding line.